src/Hazelcast.Net.Benchmarks/BasicMapOperation.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using BenchmarkDotNet.Attributes;
+using Microsoft.Extensions.Logging;
+
+namespace Hazelcast.Benchmarks
+{
+    public class BasicMapOperation
+    {
+        private IHazelcastClient _client;
+        const string _mapName = "highWay";
+        private string _key = "dearMama";
+        private string _value = "Ain't a woman alive that could take my mama's place";
+
+        //They does not work inline process. To overcome that
+        //seperate the class into a different project.
+        //details : https://benchmarkdotnet.org/articles/features/etwprofiler.html
+        //[NativeMemoryProfiler]
+        //[MemoryDiagnoser]
+        //[HardwareCounters(HardwareCounter.TotalCycles, HardwareCounter.BranchInstructions)]
+        public BasicMapOperation()
+        {
+
+        }
+
+        private string[] GenerateWords(int count, int minLength = 5, int maxLength = 10)
+        {
+            string[] words = new string[count];
+
+            for (int i = 0; i < count; i++)
+            {
+                string word = GenerateWord(minLength, maxLength);
+                words[i] = word;
+            }
+
+            return words;
+        }
```

Comment:
Not used I think, are we planning to use it in the future?

Reply:
Hold on - how come this is part of the PR about logging at all?!

Reply:
That was a left over, fixed at c5955e9a36087d0b645b845cda4133779c13a6ef